requirements.txt

```diff
@@ -16,4 +16,4 @@ python-multipart==0.0.17
 mixedbread-ai==2.2.6
 pypdf2==3.0.1
 sentry-sdk==2.19.0
-# fastembed==0.4.2
+fastembed==0.4.2
```

Comment:
Hey @moda20 👋 

Thanks for the PR 😄  But we would like to keep it optional for now. Our managed servers don't use fastembed yet. This is a better alternative for self hosting.

Reply:
@sansyrox if not used, the dependency existence shouldn't add any issues,

Reply:
@moda20 , fastembedd has pytorch as a sub dependency which slows down the boot time by lot. (read adds 3 minutes, whereas it is less than 1 minute right now)